tests/host_tools/readmem.c

```diff
@@ -0,0 +1,55 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
```

Comment:
What is the overall purpose of this application?

Reply:
Added a description of the program.